backend/doc/cli.md

```diff
@@ -32,16 +32,85 @@ This will drop the entire database, be careful!
 
 ### add a user
 
-    tracimcli user create -l "john@john@john.john" -p "superpassword"
+```
+$ tracimcli user create -h
+usage: tracimcli user create [-h] [-c CONFIG_FILE] [-d] [-e EMAIL]
+                             [-u USERNAME] [--public-name PUBLIC_NAME]
+                             [--allowed_space ALLOWED_SPACE] [--lang LANG]
+                             [-p PASSWORD] [--profile PROFILE]
+                             [--timezone TIMEZONE] [--send-email]
+
+Create user
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c CONFIG_FILE, --config CONFIG_FILE
+                        application config file to read (default:
+                        development.ini)
+  -d, --debug_mode      enable Tracim log for debug
+  -e EMAIL, --email EMAIL
+                        set user email
+  -u USERNAME, --username USERNAME
+                        set username of the user
+  --public-name PUBLIC_NAME
+                        set user public name
+  --allowed_space ALLOWED_SPACE
+                        set user allowed space in bytes
+  --lang LANG           set user lang (ISO 639 format)
+  -p PASSWORD, --password PASSWORD
+                        set user password
+  --profile PROFILE     set user profile
+  --timezone TIMEZONE   set user timezone
+  --send-email          Send mail to user
+```
+
+Example:
+
+```
+  tracimcli user create -e "john@john@john.john" -p "superpassword"
```

Comment:
Add one more exemple with more value (profile, username, etc.).
Also: is it possible to create user without show password in terminal ? (this is not directly related to this documentation I think).

Reply:
Added 2 other example + explanation about behaviors when password is not set at user creation.
About password not showed in terminal, this seems complex to do so, it may be possible to do with an interactive mecanism.Now, you can create user without password or with autogenerated password (if send-email provided) as explained in documentation update.

Reply:
Ok. thanks for your answer.